test/runtime/gbt/comm/bigBulkTran.execopts

```diff
@@ -0,0 +1,2 @@
+--doGET=true
+--doGET=false
```

Comment:
I think it'd be worth running this test at a couple different sizes. Maybe 512, 1024, 2048, and possibly 1/3 of available mem (although that will probably increase the running time significantly)

Reply:
Whether that's worthwhile would seem to depend on the purpose of the test.  If it's just a test that we're dealing with the Gem/Ari limitations properly then I think 2gb and running only on comm=ugni is sufficient.  (This was my intent.)  If we want it to be more of a general test then running it with other comm layers is appropriate, and we probably would at least want different sizes for each comm layer, depending on how their logic flowed.

Reply:
Even if we only run it for ugni, I think it's worth running at "boundary sizes" to make sure things work as expected (that said I think it's worth running for other configs too)